packages/contracts/deploy/helpers.ts

```diff
@@ -16,6 +16,15 @@ export const ENS_ADDRESSES: {[key: string]: string} = {
   goerli: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e', // aragon.eth
 };
 
+export const DAO_PERMISSION = [
```

Comment:
```suggestion
export const DAO_PERMISSIONS = [
```

Reply:
This would include changing it in other places you use !

Reply:
amended